src/developer/web-api/analytics.md

```diff
@@ -181,6 +181,33 @@ the pagination parameter `paging=true` to the URL.
 
     /api/33/dimensions/J5jldMd8OHv/items?paging=true
 
+The `/dimensions` API also provides an endpoint where the clients can get the *recomendations* for a given set of *dimensions*. For example:
+
+    /api/33/dimensions/recommendations?fields=id&dimension=dx:fbfJHSPpUQD
+
+In the example above, the client will receive back all the *Categories* that has `Data dimension` set to `true`.
+The *Categories*, in this case, will be the ones associated (through data sets and category combos) with the data element `fbfJHSPpUQD`.
+In addition, all *Organization Unit Group Set* that have `Data dimension` set to `true` will also (and always) be returned as part of the response.
+
+In other words, in a more technical approach, behind the scenes it does:
+
+1) For each data element (`dx`) provided in the URL as dimension do
+
+   `dx` -> GET category combos -> GET categories -> CHECK if `Data dimension` is true -> THEN include the `Category` in the response list.
```

Comment:
I am wondering if this goes a bit too much into the inner details/workings of the API. Could we explain it a bit simpler and more from an external perspective perhaps @maikelarabori ?

Reply:
Added another comment with alternative.